### DISCUSSION TITLE
Mastodon server name character limit

### DISCUSSION THREAD

**Discussion by sudhagar-mr-17051** [mastodon/mastodon#26051] [Q&A]
Is there any restriction in max character allowed for setting mastodon server name? What is the max character size for server name??
Need clarity little urgently, kindly help us
Thanks in advance

**Comment by ryanmio** [comment]
No Mastodon limit, but DNS standards restrict fully-qualified domain names to 253 characters, individual labels to 63 characters

**Reply by ryanmio** [reply]
@sudhagar-msr did this answer your question?
